index.js

```diff
@@ -1,21 +1,42 @@
+/*jshint node: true */
+'use strict';
+
 var path = require('path');
+var mergeTrees = require('broccoli-merge-trees');
+var Funnel = require('broccoli-funnel');
 
 module.exports = {
   name: 'ember-data-factory-guy',
 
+  treeForVendor: function() {
```

Comment:
OK I see what you mean now. I'll try to find some time to play around with the issue and come to a solution.